pom.xml

```diff
@@ -400,6 +400,7 @@
                                     <exclude>LICENSE.txt</exclude>
                                     <exclude>**/.dontdelete</exclude>
                                     <exclude>docs/**</exclude>
+                                    <exclude>**/*.adoc</exclude>
```

Comment:
Is there a reason we don't add the license or copyright header to the adoc files?